addons/api/tests/unit/abilities/session-test.js

```diff
@@ -108,4 +108,74 @@ module('Unit | Ability | session', function (hooks) {
 
     assert.true(canService.can('read session', session));
   });
+
+  test('can cancel a session that is active', function (assert) {
+    assert.expect(1);
+
+    const session = store.createRecord('session', {
+      authorized_actions: ['cancel'],
```

Comment:
same as above, should we add test cases for `read:self` and `cancel:self`?

Reply:
I did add test cases for them